server/tasks/change_gating.py

```
@@ -0,0 +1,815 @@
+            row = cur.fetchone()
+            if row is not None and row[0] is not None:
+                return "suspended"
+
+            if not set_rls_context(cur, conn, user_id, log_prefix="[ChangeGating]"):
+                # Org resolution failing is a (likely transient) error, NOT
+                # proof of non-enrollment — raise so the task retries instead
+                # of silently skipping the review.
+                raise RuntimeError(
+                    f"RLS context unavailable for user {user_id} — cannot "
+                    "verify change-gating enrollment"
+                )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)